external-docs/docs/guides/inspect-api/example3.md

```diff
@@ -0,0 +1,196 @@
+
+>>> fa = FactorAnalyzer(n_factors=3, rotation=None)
+>>> fa.fit(pivoted_df)
+>>> ev, v = fa.get_eigenvalues()
+>>> ev
+array([1.77598209, 0.86422349, 0.35979442])
+
+>>> fig = px.line(x=range(1, pivoted_df.shape[1]+1), y=ev)
+>>> fig.show()
+```
+
+![Eigenvalue plot](../../images/eigenvalues.png)
+
+```python
+>>> loadings = fa.loadings_
+>>> loadings
+array([[ 0.89912209,  0.02503715,  0.        ],
+       [-0.60131768,  0.39154271,  0.        ],
+       [ 0.5034576 ,  0.42293552,  0.        ]])
+```
+
+## Another example: Generalised Linear Model
+
+This is another example of how information can be retrieved from a CSVW with the help of the Inspect API. The purpose of this example is to highlight how Inspector objects and the information they give access to can streamline exploration of data and avoid the requirement of any thorough knowledge of the data set or its details in order to retrieve information to use in analysis.
+
+Here, we will be using the Inspect API to acquire all the information we need to then create a basic GLM (Generalised Linear Model) using the statsmodels library.
+
+Say we want to make a model on a data set where we want the response variable to be modelled on the observation values from our input data set, and we want it to depend on the values contained within a time period column, i.e. dimension column.
+
+```python
+>>> gross_pay_inspector = Inspector('gross-median-weekly-pay.csv-metadata.json')
+>>> gross_pay_table = gross_pay_inspector.tables[0]
+>>> gross_pay_columns = gross_pay_table.columns
+```
+
+After loading the Inspector object, we can get the information we need from it. Refer to the previous example [Exploring data](./example3.md/#exploring-the-data) for accessing information using an Inspector, e.g. the data set's primary table and then the columns. It will work the same way, all we need to do is provide a valid input file.
+
+Then we can get the dimensions columns from the data set. Note for an analysis like this, we assume there is only one observations column in the data set with an associated dimension. Otherwise, you would obviously need to know which set of observations you want to analyse and specify it.
+
+The previous example's [Formatting data](./example3.md/#formatting-the-data) section shows how to acquire columns of a specific type using the Inspector object. We can do the same here to get the observation and dimension column titles.
+
+```python
+>>> dimension_col_title = [title for title, column in gross_pay_columns.items() if isinstance(column, DimensionColumn)]
+>>> value_col_title = [title for title, column in gross_pay_columns.items() if isinstance(column, ObservationColumn)]
+```
+
+We have acquired the titles of the observations column and the dimensions column from our data set. Now, we can input this information into the GLM creation functions. The rest of the code is just the creation of a basic GLM.
+
+```python
+>>> glm_formula = f"{value_col_title} ~ {dimension_col_title}"
```

Comment:
Won't this only work if there's just one dimension? What happens when there is more than one dimension?